### PR TITLE
Added --beacon.api flags to enable experimental beacon api.

### PR DESCRIPTION
**Pull request by Giulio2002**
Make it so that erigon can the enable beacon api.

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -862,6 +863,41 @@ var (
...
+		Usage: "Sets the seconds for a read time out in the beacon api",
+		Value: 5,
+	}
+	BeaconApiWriteTimeoutFlag = cli.Uint64Flag{
+		Name:  "beacon.api.write.timeout",
+		Usage: "Sets the seconds for a write time out in the beacon api",
+		Value: 5,
+	}
+	BeaconApiIdleTimeoutFlag = cli.Uint64Flag{
+		Name:  "beacon.api.ide.timeout",
```
Comment (AskAlexSharov) [inline cmd/utils/flags.go:887]: `idle`?

File: cmd/utils/flags.go [hunk 2]
```diff
@@ -862,6 +863,41 @@ var (
...
+	}
+	BeaconApiAddrFlag = cli.StringFlag{
+		Name:  "beacon.api.addr",
+		Usage: "sets the host to listen for beacon api requests",
+		Value: "localhost",
+	}
+	BeaconApiPortFlag = cli.UintFlag{
+		Name:  "beacon.api.port",
+		Usage: "sets the port to listen for beacon api requests",
+		Value: 5555,
```
Comment (AskAlexSharov) [inline cmd/utils/flags.go:899]: add new port to readme ports section

File: cmd/utils/flags.go (hunk 1 above)
Comment (Giulio2002) [reply cmd/utils/flags.go:887]: it is used in ListenAndServe, rpcdaemon has also a counterpart

File: cmd/utils/flags.go (hunk 2 above)
Comment (Giulio2002) [reply cmd/utils/flags.go:899]: mh sure, this is experimental for now btw

**Pull request merged**